benchmarks/util.py

```diff
@@ -50,13 +50,13 @@ def reset_rng_state(benchmark_experiment=None):
   torch.manual_seed(1337)
   random.seed(1337)
   np.random.seed(1337)
-  if benchmark_experiment is not None and benchmark_experiment.xla is not None and benchmark_experiment.torch_xla2 is not None:
+  if benchmark_experiment is not None and benchmark_experiment.xla is not None and benchmark_experiment.torch_xla2 is None:
```

Comment:
if we don't use torch_xla2, we don't need to do `xm.set_rng_state(1337, str(device))`?

Reply:
I am using `benchmark_experiment.torch_xla2 is None` in the condition. If we use torch_xla2, we don't set the rng state. 

We will work on a solution to set rng state in jax device later.

Reply:
SG. Better to add a TODO